packages/react-focus-management/src/FocusManagementProvider.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { useMergedRefs } from '@fluentui/react-hooks';
+import { makeMergeProps } from '@fluentui/react-utils';
+import { getCurrentAbilityHelpers, createAbilityHelpers, Types as AHTypes } from 'ability-helpers';
+import { internal__FocusManagementContext, FocusManagementContextValue } from './focusManagementContext';
+
+type Dir = 'ltr' | 'rtl';
+
+export interface FocusManagementProvideProps extends React.HTMLAttributes<HTMLElement> {
+  dir?: Dir;
+
+  window?: Window;
```

Comment:
https://github.com/microsoft/fluentui/blob/96f5ff4e63e7c2235f6b180e2596385ed8775f80/packages/react-provider/src/FluentProvider.tsx#L12-L13

`FluentProvider` uses `document`, can we align there?

Reply:
I don't think so... AH uses window quite heavily internally, and there's also the cross origin scenarios that Teams uses.

Reply:
switched to document.defaultView and added guards